apps/dashboard/web/polaris_web/web/src/apps/dashboard/pages/settings/test_library/api.js

```diff
@@ -0,0 +1,35 @@
+import request from "@/util/request"
+
+const testLibraryRequests = {
+    addTestLibrary(data) {
+        return request({
+            url: '/api/addTestLibrary',
+            method: 'post',
+            data: data
+        })
+    },
+    removeTestLibrary(data) {
```

Comment:
Taking `data` as input argument isn't a good practice. You should add the exact variables you need. Example 

```
   removeTestLibrary({repository, branch}) {
        return request({
            url: '/api/addTestLibrary',
            method: 'post',
            data: {repository, branch}
        })
    },

```